projects/elements/src/lib/lazy-elements/lazy-elements.module.ts

```diff
@@ -69,3 +88,14 @@ export class LazyElementsModule {
 export interface LazyElementModuleOptions {
   elementConfigs?: ElementConfig[];
 }
+
+export interface LazyElementRootOptions {
```

Comment:
So we have `forFeature(LazyElementModuleOptions )` so we should have `forRoot(LazyElementModuleRootOptions )` and NOT `forRoot(LazyElementRootOptions )`.

That way it will be consistent and easy to follow / understand for the users...